### PR TITLE
Dynamic event allocate update

### PR DESCRIPTION
**Pull request by juhhei01**
Changed Event allocate to temporary area.

Monitor dynamic allocated peak andf added API for read peak value.

### CODE REVIEW

**Comment by juhhei01** [comment]
@AnttiKauppila and @JarkkoPaso Please review this.

File: source/event.c [hunk 1]
```diff
@@ -137,10 +137,24 @@ int8_t eventOS_event_send(arm_event_s *event)
     return -1;
 }
 
+static uint16_t event_dynmaic_allocated_peak = 0;
```
Comment (JarkkoPaso) [inline source/event.c]: dynmaic

Comment (juhhei01) [reply source/event.c]: fixed

**Comment by juhhei01** [comment]
@SeppoTakalo Good point.

**Pull request merged**